tests/installation/addon_products_via_SCC_yast2.pm

```diff
@@ -35,14 +36,13 @@ sub test_setup {
         # Define proxy SCC
         assert_script_run "echo \"@addon_proxy.proxy.scc.suse.de\" > /etc/SUSEConnect";
     }
-    # Disable screensaver, fails sometimes on SLE12, so keep it as before without assertion
-    script_run "gsettings set org.gnome.desktop.session idle-delay 0";
+    turn_off_gnome_screensaver;
```

Comment:
you're replacing script_run with type_string, i.e. no check if the command is finished before continueing

please revert